### PR TITLE
docs(domains): added how-to renew domain

### PR DESCRIPTION
**Pull request by nedaehm**
### Your checklist for this pull request

- [x] Check that the commit messages match our requested structure.
- [x] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description
Please describe your pull request.

### CODE REVIEW

File: network/domains-and-dns/how-to/renew-domain.mdx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  paragraph: This page explains how to renew a domain name with Scaleway Domains and DNS.
+tags: renew domain dns
+dates:
+  validation: 2023-03-07
+  posted: 2023-03-07
+categories:
+  - network
+---
+
+This page shows you ho to renew your domain name.
```
Comment (farhakennar) [inline network/domains-and-dns/how-to/renew-domain.mdx:16]:
```suggestion
This page shows you how to renew your domain name.
```

File: network/domains-and-dns/how-to/renew-domain.mdx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+</Message>
+
+
+1. Click **Domains and DNS** in the **Network** section of the [Scaleway console](https://console.scaleway.com) side menu. The list of your domains displays.
+2. Click the domain you want to renew. Your domain's overview page displays.
+3. Click **Renew** under **Expiration date**. A pop-up displays.
+4. Choose your registration period in the drop-down list.
+
+    <Message type="note">
+     You can renew your domain for one to nine years with a maximum domain name registration period of 10 years.
```
Comment (farhakennar) [inline network/domains-and-dns/how-to/renew-domain.mdx:33]:
```suggestion
     You can renew your domain for up to nine years with a maximum domain name registration period of 10 years.
```
Pas sûre d'avoir compris, l'idée c'est qu'on peut renouveler son domaine un maximum de 9 ans ?

File: network/domains-and-dns/how-to/renew-domain.mdx [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    </Message>
+5. Check the box to confirm you understand your domain will be renewed and that you will be debited the price that displays.
```
Comment (farhakennar) [inline network/domains-and-dns/how-to/renew-domain.mdx:35]:
```suggestion
5. Check the box to confirm you understand your domain will be renewed and that you will be debited the displayed price.
```

**Review by farhakennar — APPROVED** [review]

File: network/domains-and-dns/how-to/renew-domain.mdx [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+</Message>
+
+
+1. Click **Domains and DNS** in the **Network** section of the [Scaleway console](https://console.scaleway.com) side menu. The list of your domains displays.
+2. Click the domain you want to renew. Your domain's overview page displays.
+3. Click **Renew** under **Expiration date**. A pop-up displays.
+4. Choose your registration period in the drop-down list.
+
+    <Message type="note">
+     You can renew your domain for one to nine years with a maximum domain name registration period of 10 years.
```
Comment (nedaehm) [inline network/domains-and-dns/how-to/renew-domain.mdx:33]:
```suggestion
     You can renew your domain for up to 9 years with a maximum domain name registration period of 10 years.
```

**Pull request merged**